src/site/_includes/components/Assessment.js

```diff
@@ -125,8 +123,8 @@ function responseTemplate(response) {
 
   if (
     response.columns &&
-    response.columns != true &&
-    response.columns != false
+    response.columns !== true &&
```

Comment:
@robdodson @samthor `typeof response.columns !== 'boolean'` instead of checking if not true and not false?

Reply:
hm sure. I think this code was reviewed in a rush when Michael F. was leaving. Should have caught this.